### PR TITLE
Rewrite page

### PR DESCRIPTION
**Pull request by JasperDhaene**
Closes Issue #16

### CODE REVIEW

File: app/assets/stylesheets/custom/ows_page.less [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/*******************************
+           .ows_page
```
[the remark this answers is not in the document]
Comment (Dromin) [reply app/assets/stylesheets/custom/ows_page.less]: My intention when I named those classes was to save the `ows_` prefix for code that has to do with slide / course rendering; that way we can be sure that there are never any naming conflicts with styles for the rest of the app. So preferably the page layout component should just be called `.page` or maybe `.page-layout` or something.

File: app/pages/Page.js [hunk 2]
```diff
@@ -15,17 +15,19 @@ type Props = TranslatorProps & PassedProps;
...
-        <Grid.Column stretched={true} width={14}>
-          <Container>
-            {props.children}
-          </Container>
-        </Grid.Column>
-      </Grid>
-    </div>
+      <div className="page-layout__grid">
+        <Grid stretched={true}>
+          <Grid.Column width={1} />
```
Comment (Dromin) [inline app/pages/Page.js]: Page is still not completely centered at small viewport sizes, probably because of this unnecessary column?

Comment (JasperDhaene) [reply app/pages/Page.js]: Looks centered to me. I put it there initially for some padding on the left, but I can also do that with LESS now.

Comment (Dromin) [reply app/pages/Page.js]: One sec, let me take a screenshot :)

Comment (Dromin) [reply app/pages/Page.js]: not-centered

^ Happens at the smallest fluid size, right before the page becomes fixed-width.

**Pull request closed without merging**